### PR TITLE
If submission deadline has passed, return error

### LINKED ISSUE #37: Implement proposal due by

### PR DESCRIPTION
**Pull request by trizin**
Closes #37

### CODE REVIEW

File: backend/routes/project.js [hunk 1]
```diff
@@ -96,12 +97,22 @@ router.post(
...
-    const currentRound = await getCurrentRoundNumber();
+    const currentRound = await getCurrentRound();
+    const currentRoundNumber = currentRound.fields["Round"];
+    const currentRoundSubmissionDeadline =
+      currentRound.fields["Proposals Due By"];
+
+    // if submission deadline has passed, return error
+    if (Date.now() > new Date(currentRoundSubmissionDeadline).getTime()) {
+      return res.status(400).json({
+        error: "Submission deadline for this round has passed",
```
Comment (idiom-bytes) [inline backend/routes/project.js:108]: Great!

File: backend/utils/airtable/utils.js [hunk 2]
```diff
@@ -70,6 +71,18 @@ async function getCurrentRoundNumber() {
...
+/**
+ * Returns the current round
+ * @return {object} current round
+ */
+async function getCurrentRound() {
+  const nowDateString = new Date().toISOString();
+  const roundParameters = await _getFundingRoundsSelectQuery(
+    `AND({Voting Starts} > "${nowDateString}", "true")`
+  );
+  return roundParameters ? roundParameters[0] : -1;
```
Comment (idiom-bytes) [inline backend/utils/airtable/utils.js:83]: Will the results always be ordered by the most immediate round?

I'm guessing this remains correct as long as the order inside Airtable is maintained. 

Great!

**Comment by idiom-bytes** [comment]
Reviewed PR, looks great. Thanks for adding this check!

File: backend/utils/airtable/utils.js (hunk 2 above)
Comment (trizin) [reply backend/utils/airtable/utils.js:83]: Thanks for pointing this out!

Comment (trizin) [reply backend/utils/airtable/utils.js]: How about this?
https://github.com/oceanprotocol/oceandao-proposal-portal/pull/45/files#diff-ef56c4c46f494e477b24b6e35c0fc9c1206a260c3e8653c3030da2b480186eb0R78

File: backend/utils/airtable/utils.js [hunk 3]
```diff
@@ -58,16 +58,26 @@ async function getProjectUsdLimit(projectName) {
...
+
+/**
+ * Returns the current round
+ * @return {object} current round
+ */
+async function getCurrentRound() {
   const nowDateString = new Date().toISOString();
   const roundParameters = await _getFundingRoundsSelectQuery(
-    `AND({Voting Starts} > "${nowDateString}", "true")`
+    `AND({Start Date} <= "${nowDateString}",{Voting Starts} > "${nowDateString}", "true")`
```
Comment (idiom-bytes) [inline backend/utils/airtable/utils.js:78]: Looks good!

**Pull request merged**